builtin/providers/aws/resource_aws_api_gateway_method.go

```diff
@@ -134,6 +140,14 @@ func resourceAwsApiGatewayMethodUpdate(d *schema.ResourceData, meta interface{})
 		operations = append(operations, expandApiGatewayRequestResponseModelOperations(d, "request_models", "requestModels")...)
 	}
 
+	if d.HasChange("request_parameters_in_json") {
+		ops, err := expandApiGatewayMethodResponseParametersJSONOperations(d, "request_parameters_in_json", "requestParameters")
```

Comment:
I think we should rename the function to make the purpose more accurate:
`expandApiGatewayMethodResponseParametersJSONOperations` => `expandApiGatewayMethodParametersJSONOperations`

(i.e. it is now being used for both _response_ and _request_ parameters so it should be called accordingly)